common/styles/utilities/_root-scope-classes.scss

```diff
@@ -253,6 +253,10 @@
   float: left;
 }
 
+.rotate-r-8 {
```

Comment:
Is this one-off-y enough to be inline?

I guess you don't get the vendor prefixes.